examples/TUTORIAL.md

```diff
@@ -0,0 +1,253 @@
+Creating a basic S2I builder image
+---
+
+### Getting started
+
+The builder image that you specify must contain all required libraries and tools to be able to build and run the intended application.
+This tutorial uses the [openshift/base-centos7](https://github.com/openshift/sti-base) image that has, among other pre-installed libraries and tools, the epel repository added which allows us to install Nginx, since it does come with the standard centos 7 repositories.
```

Comment:
i'd rather see an example that starts from just centos, rather than our sti-base.  Again we want barebones and don't want them to have to go dig into that image to see what magic we're doing there.  Everything they need to know should be provided in the files that are in this tutorial, not inherited from elsewhere.